seqr/views/apis/anvil_workspace_api.py

```diff
@@ -31,6 +32,12 @@
 
 BLOCK_SIZE = 65536
 
+ANVIL_LOADING_EMAIL_DATE = '2021-12-13'
```

Comment:
You may also want to add an ending date.

Reply:
as explained above, no because we don't know the end date yet